### PR TITLE
Changed spec file exclude unit testing

### PR DESCRIPTION
**Pull request by gnehapk**
tendrl-github-id: #791

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -273,4 +273,4 @@ gulp.task("release", ["common"], function (done) {
 });
 
 //default task is release
```
Comment (r0h4n) [inline gulpfile.js]: change this comment to "default task is common"

**Review by r0h4n — CHANGES_REQUESTED** [review]

File: gulpfile.js (hunk 1 above)
Comment (gnehapk) [reply gulpfile.js]: @r0h4n ack

File: .travis.yml [hunk 2]
```diff
@@ -5,4 +5,4 @@ before_install:
   - npm install -g gulp
   - npm install
 script:
-  - gulp
+  - gulp release
```
Comment (TimothyAsirJeyasing) [inline .travis.yml:8]: So this will add unit test now to travis right ?

Comment (gnehapk) [reply .travis.yml:8]: @TimothyAsir Yes. gulp release task includes unit testing as well.

**Review by r0h4n — APPROVED** [review]

**Pull request merged**